packages/gitbook-docs/src/markdoc/attributes.ts

```diff
@@ -28,11 +28,16 @@ export class LinkAttr {
   readonly transform = (value: string | null, { variables }: Config) => {
     // TODO: this cast will be removed when we can pass a custom type instead of Config
     const parseContentConfig = variables as ParseContentConfig;
-    // Link to other spaces start with http://localhost:5000
+    // Link to other spaces when start with http://localhost:5000  
     const linkToSpacesRegex = new RegExp(
       '^http:\\/\\/localhost:5000(\\/o\\/[\\w]*)?\\/s\\/(.*?)\\/?$',
       'g'
     );
+    // Link to other spaces when start with http://127.0.0.1:5000
+    const linkToSpacesRegex_1 = new RegExp(
```

Comment:
Using this expression  `'^http://(?:localhost|127\\.0\\.0\\.1):5000(?:/o/([\\w-]+))?/s/([\\w-]+)/?$',` , it is possible to parse both paths (localhost and 127.0.0.1) with a single variable, but it doesn't pass the testing phase